app/helpers/brexit_checker_helper.rb

```diff
@@ -70,4 +70,32 @@ def notification_email_link(non_tracked_url, notification)
 
     url.to_s
   end
+
+  def email_link_label(actions)
```

Comment:
Since all helper methods are global, we should scope these generic methods to avoid clashes.

Reply:
ah great point, I'd guessed they were more specifically scoped.

Humm `brexit_results_title` and so on?

Reply:
Cool, amended that too